assignment1.txt

```diff
@@ -0,0 +1,11 @@
+Clarke Littlejohn
+
+1) The significance of return 0 is that it allows for the person to see if the code compiled correctly.
+ An integer needs to be returned if the function is delcared "int function".
+
+2) The issue in the code is that the operator is wrong; the operator should be "%f" instead "%d".
```

Comment:
You're right, but you're missing a couple of errors - check to see if you can find them.